tests fail python 3.5

As far as I can tell your CI tests don't do Python 3.5 yet.

Looks like the error generated from python has changed.

```
=================================== FAILURES ===================================
____ TestUserSetter.test_calling_user_fails_when_attribute_error_is_raised _____
tests/test_request.py:269: in test_calling_user_fails_when_attribute_error_is_raised
    self.assertEqual(str(error), "'module' object has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'")
E   AssertionError: "module 'rest_framework' has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'" != "'module' object has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'"
E   - module 'rest_framework' has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'
E   ?       - ^ ^ -----------
E   + 'module' object has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'
E   ? +       ^^^^ ^
========= 1 failed, 683 passed, 52 skipped, 2 warnings in 4.09 seconds =========
```

--- a/tests/test_request.py
+++ b/tests/test_request.py
@@ -186,7 +186,10 @@ class TestUserSetter(TestCase):
         try:
             self.request.user
         except AttributeError as error:
-            self.assertEqual(str(error), "'module' object has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'")
+            assert str(error) in (
+                "'module' object has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'",  # Python < 3.5
+                "module 'rest_framework' has no attribute 'MISSPELLED_NAME_THAT_DOESNT_EXIST'",  # Python >= 3.5
+            )
         else:
             assert False, 'AttributeError not raised'
 

